sandbox.sh

```diff
@@ -21,32 +18,39 @@ export PATH
 
 [ "$USE_NIX" ] && dune build @install
 
+# Use tezos-client to interact with the node.
+# See https://tezos.gitlab.io/introduction/howtouse.html#client
 tezos-client() {
   docker exec -t deku_flextesa tezos-client "$@"
 }
 
+# Necessary to write tezos smart contracts
 ligo() {
   docker run --rm -v "$PWD":"$PWD" -w "$PWD" ligolang/ligo:0.28.0 "$@"
 }
 
-if [ $mode  = "docker" ]
-then
+if [ $mode = "docker" ]; then
   RPC_NODE=http://flextesa:20000
 else
   RPC_NODE=http://localhost:20000
 fi
 
+# Tezos secret key?
```

Comment:
And what its purpose?

Reply:
For signing tezos operations. You have to give the secret key to the deku node as part of configuration.